src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((block) => block.trim())
+    .filter((block) => block.includes(':'))
+    .reduce((cssProperties, block) => {
+      const [key, value] = block
+        .split(':')
+        .map((splitBlock) => splitBlock.trim());
```

Comment:
Consider using destructuring directly in the parameters of the map callback to enhance clarity and reduce the need for additional variable assignment inside the block. For example: `.map(block => block.split(':').map(part => part.trim()))` can be refactored to `.map(block => { const [key, value] = block.split(':').map(part => part.trim()); ... })`.